go-controller/pkg/clustermanager/endpointslice_mirror_controller_test.go

```diff
@@ -0,0 +1,445 @@
+package clustermanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
```

Comment:
maybe consider using ginkgo v2 here ?

Reply:
I tried to but `GenerateNad` is a part of the testing package which pulls in ginkgo v1.
This breaks on init:
```
panic: ... flag redefined: ginkgo.seed
```
Cannot modify the testing package as it is used by other ginkgo v1 tests.

:(

Reply:
I think that moving all unit tests to ginkgo v2 makes more sense. wdyt?

Reply:
of course :) 

I just thought it would be free. 
Thanks for explaining.

Reply:
FYI: https://github.com/ovn-org/ovn-kubernetes/pull/4520 lets see how it behaves :)